projects/managed-hubs/support.md

```diff
@@ -1,6 +1,105 @@
+
+The current support team is listed in an [ongoing GitHub issue](https://github.com/2i2c-org/team-compass/issues/294).
+In the sprint meeting, we should edit this issue to add/remove Support Stewards as necessary.
+
+```{button-link} https://github.com/2i2c-org/team-compass/issues/294
+Support team issue
+```
+
+## Responsibilities and expectations
+
+These are some major responsibilities and goals of the support team:
+
+- Be the main point of contact for all support tickets regardless of the community
+- Communicate with the person opening a support ticket, to keep them in the loop
+- Create backlog issues as-needed
+- Route support issues to the engineering team in order to make the changes necessary to resolve the issue.
+- During events, be extra attentive to support channels in case a change is needed.
+
+:::{note}
+We should try to respond to all support-related communications **within one working day**.
+Even if the support request is not resolved, others appreciate when we keep them in-the-loop about what we're doing.
+:::
+
+## Tracking support items
+
+### FreshDesk requests
+
+For bugs, hub changes, improvements, and general questions, we use the [e-mail address `support@2i2c.org`](mailto:support@2i2c.org).
+This is connected with [the 2i2c FreshDesk account](https://2i2c.freshdesk.com/), and is a central place for all support-related requests.
+
+Many requests may be immediately resolved via a quick action or response to the community representative.
+However, if a request requires consultation with the team, open a backlog issue.
+
+(support:issues)=
+### Support issues
+
+If resolving a support issue requiries ongoing or concerted owrk, open a backlog issue tagged with the {guilabel}`support` label.
+
+Support backlog items should:
+
+- Be placed in our [Sprint Board](coordination:sprint-board).
+- Include a reference to any FreshDesk tickets if they exist.
+- Be prioritized over other backlog items for that sprint.
+
+## Communication channels
+
+We use two Slack channels to discuss support-related questions on the 2i2c hub:
+
+- The [#support-freshdesk](https://2i2c.slack.com/archives/C028WU9PFBN) is for discussing support tickets on Freshdesk.
+  Open a Slack thread for each support ticket, to have conversation around it amongst the team.
+  This channel is private because there’s an assumption of privacy when we discuss community-specific support matters.
+- The [#hub-support](https://2i2c.slack.com/archives/C01DB2JRP8W) channel is for discussing more general support-related issues relating to operating the 2i2c Hubs, and not not related to a specific FreshDesk ticket.
+  This channel is public, so can be useful if we wish to invite conversation from non 2i2c team members.
+
+## Lifecycle of a support request
+
+- When you receive a ticket, communicate that you have received it and note the next steps you’ll follow to resolve it (even if this just means investigating because you're not sure what is wrong).
```

Comment:
```suggestion
- We should try to respond to all support-related communications within one working day.
- When you receive a ticket, communicate that you have received it and note the next steps you’ll follow to resolve it (even if this just means investigating because you're not sure what is wrong).
```